Added reference kernel fallback for the fully_connected kernel

lamarrr opened the pull request:
I added the reference kernel fallback for the fully_connected kernel's EvalQuantizedInt8 routine.

lamarrr committed:
Added reference kernel fallback for the fully_connected kernel

I added the reference kernel fallback for the fully_connected kernel's EvalQuantizedInt8 routine.

freddan80 commented on tensorflow/lite/experimental/micro/kernels/cmsis-nn/fully_connected.cc:
```diff
@@ -111,7 +111,23 @@ TfLiteStatus EvalQuantizedInt8(TfLiteContext* context, TfLiteNode* node,
       ARM_MATH_SUCCESS);
 #else
-#error ARM_MATH_DSP and ARM_MATH_LOOPUNROLL must be set
+
```
Actually, it would be good for the user to be notified that there's no optimization for this target. You can add a directive for that. For example:
#pragma message("CMSIS-NN optimization not available for this target. Using ref kernel.")

lamarrr replied:
done!

lamarrr committed:
added CMSIS-NN optimization error message

added directive to notify the user of unavailability of CMSIS-NN optimization for the compilation target

lamarrr committed:
Update fully_connected.cc

rthadur commented:
gentle ping @petewarden for review ?
